vnext/Desktop/ABI/ReactNativeHost.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include <winrt/Windows.Foundation.Collections.h>
+
+#include "Microsoft.ReactNative.ReactNativeHost.g.h"
+
+namespace winrt::Microsoft::ReactNative::implementation {
+
+struct ReactNativeHost : ReactNativeHostT<ReactNativeHost> {
```

Comment:
>ReactNativeHost [](start = 7, length = 15)

It would be nice to reuse files from the Microsoft.ReactNative project in future.

Reply:
I will attempt this with the next change.

---
In reply to: [423289186](https://github.com/microsoft/react-native-windows/pull/4848#discussion_r423289186) [](ancestors = 423289186)